hexl/ntt/ntt-radix-2.cpp

```diff
@@ -293,6 +386,10 @@ void InverseTransformFromBitReverseRadix2(
     t <<= 1;
   }
 
+  if(result != operand && n == 2){
```

Comment:
Why is this needed?

Reply:
I am using the first pass (t == 1) to copy _operand_ into _result_ but it seems like in some (unit tests) cases _m_ is too short it only needs to do the final stage butterfly, meaning it doesn't enter the previous _for_ where it would normally do the copy, so I had to add those lines.

Reply:
Makes sense; perhaps adding a quick comment